src/com/nutrons/steamworks/Drivetrain.java

```diff
@@ -0,0 +1,33 @@
+package com.nutrons.steamworks;
+
+import static com.nutrons.framework.util.FlowOperators.deadband;
+import static io.reactivex.Flowable.combineLatest;
+
+import com.nutrons.framework.Subsystem;
+import com.nutrons.framework.controllers.ControllerEvent;
+import com.nutrons.framework.controllers.RunAtPowerEvent;
+import io.reactivex.Flowable;
+import io.reactivex.functions.Consumer;
+
+public class Drivetrain implements Subsystem {
+  private final Flowable<Double> throttle;
+  private final Flowable<Double> yaw;
+  private final Consumer<ControllerEvent> leftDrive;
+  private final Consumer<ControllerEvent> rightDrive;
+  private double coeff = 1.0;
+
+  // Does cheezy drive.
```

Comment:
This comment is not JavaDoc, which would help people understand this code, nor is it an implementation comment.  Either eliminate it or make it real documentation.